.coafiles

```diff
@@ -1,4 +1,4 @@
 [c]
-files = .coafiles
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `c`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpgqmczvwz/.coafiles
+++ b/tmp/tmpgqmczvwz/.coafiles
@@ -1,4 +1,4 @@
 [c]
-files = .coafiles 
+files = .coafiles
 bears = SpaceConsistencyBear
 use_spaces = yeah 
```